test/nlp425/geo_analyzer.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * 
+ */
+package nlp425;
+
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.Test;
+
+import edu.pugetsound.mathcs.nlp.lang.Utterance;
+import edu.pugetsound.nlp.geoanalyzer.GeoAnalyzer;
+
+/**
+ * @author brian
+ *
+ */
+public class geo_analyzer {
+	
+	
+	@Test
+	public void EnsureAnalyzeReturnsUtterance() {
+		GeoAnalyzer geo_test = new GeoAnalyzer(".");
+		assert(geo_test.geoAnalyze("This is a sentance") instanceof Utterance);
+	}
+	
+	@Test
+	public void TestAnalysisForExceptions() {
```

Comment:
You can merge this unit test with the one above since functionally they are the same.